histoqc/ClassificationModule.py

```diff
@@ -183,6 +183,13 @@ def byExampleWithFeatures(s, params):
 
                 if nsamples_per_example != -1: #sub sambling required
                     nitems = nsamples_per_example if nsamples_per_example > 1 else int(mask.shape[0]*nsamples_per_example)
+                    
+                    # get random seed from config variable
+                    seed = int(params.get("seed", 0))
```

Comment:
0 is technically a valid value for a seed. Can you change the default to another falsy value like None or False, and check for that value on line 190?